### PR TITLE
refactor(core): cleanup code for unused components

### PR DESCRIPTION
**Pull request by jayzhang**

### CODE REVIEW

**Comment by blackchoey** [comment]
This PR decreased the overall code coverage by **1.32%**, can you create a bug as follow up of code coverage improvement? I'm not sure whether the removed test cases also tested some other source code, and removing the test cases causes some code no longer being tested.

cc @LongOddCode , please share your comments if you have

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Comment by jayzhang** [comment]
> 



> This PR decreased the overall code coverage by **1.32%**, can you create a bug as follow up of code coverage improvement? I'm not sure whether the removed test cases also tested some other source code, and removing the test cases causes some code no longer being tested.
> 
> cc @LongOddCode , please share your comments if you have

The removed test cases are for v2 code.

**Review by nliu-ms — APPROVED** [review]

**Review by Siglud — APPROVED** [review]

File: packages/fx-core/src/component/resource/aadApp/aadApp.ts [hunk 1]
```diff
@@ -191,11 +183,6 @@ export class AadApp implements CloudResource {
     return res;
   }
 
-  @hooks([
```
Comment (KennethBWSong) [inline packages/fx-core/src/component/resource/aadApp/aadApp.ts:194]: We still will call collaboration/buildAadManifest for non-lifecycle commands. Will this change have any influence?
cc @wenytang-ms

Comment (jayzhang) [reply packages/fx-core/src/component/resource/aadApp/aadApp.ts:194]: No, because FxCore already has a middleware to replace this deleted one.

**Review by KennethBWSong — APPROVED** [review]

**Review by xzf0587 — APPROVED** [review]

File: packages/fx-core/src/component/resource/aadApp/aadApp.ts (hunk 1 above)
Comment (jayzhang) [reply packages/fx-core/src/component/resource/aadApp/aadApp.ts:194]: I will make up for the telemetry by addStartAndEndTelemetry(actionName, actionName) middleware in another PR.

**Review by hund030 — APPROVED** [review]

**Review by SLdragon — APPROVED** [review]

**Review by sffamily — APPROVED** [review]

**Pull request merged**